amrit-local-setup/loaddummydata.sh

```
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# MySQL connection details
+HOST="127.0.0.1"
+PORT="3307"
+USER="root"
+PASSWORD="1234"
+
+# Path to the extracted SQL files
+SQL_DIR=<PATH to the  extracted files directory>
```

Shellcheck (0.10.0)

[error] Couldn't parse this variable assignment. Fix to allow more checks.

(SC1073)